examples/source_examples/veeva_vault_objects/configuration.json

```diff
@@ -0,0 +1,5 @@
+{
+    "vaultDNS": "<YOUR_VEEVA_VAULT_DNS>.veevavault.com",
```

Comment:
```suggestion
    "subdomain": "<YOUR_VEEVA_VAULT_SUB_DOMAIN>.veevavault.com",
```

Reply:
please change your python code accordingly

Reply:
Code has been updated to reflect this change.